### ISSUE TITLE
[HOLD for payment 2023-04-26] [HOLD for payment 2023-04-21] [$1000] Special html characters like &nbsp  are not escaped in name and can bug out the UI

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#15131]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed

1. Change your name to &nbsp
2. Notice that all the reports and IOUs just have a blank space

## Expected Result
There should be no blank space and the &nbsp should be shown normally

## Actual Result
There is blank space for the name.

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?

- [x] MacOS / Chrome / Safari
- [x] MacOS / Desktop

**Version Number:** 1.2.71-1
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/218810285-ce35c1f8-3ad5-4df9-b43f-93f53c3a651d.mp4

https://user-images.githubusercontent.com/43996225/218810351-60b70e41-90cf-4bd0-8a0f-50c621b08ee0.mov

**Expensify/Expensify Issue URL:**
**Issue reported by:** @esh-g 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1676317301468499

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01823be2bad9582bf7</li>
        <li>Upwork Job ID: 1625814162742865920</li>
        <li>Last Price Increase: 2023-02-15</li>
    </ul>
</details>

**Comment by MelvinBot** [comment]
Triggered auto assignment to @flaviadefaria (`Bug`), see https://stackoverflow.com/c/expensify/questions/14418 for more details.

**Comment by MelvinBot** [comment]
### Bug0 Triage Checklist ([Main S/O](https://stackoverflowteams.com/c/expensify/questions/14418))

- [ ] This "bug" occurs on a supported platform (ensure `Platforms` in OP are ✅)
- [ ] This bug is not a duplicate report (check [E/App issues](https://github.com/Expensify/App/issues) and [#expensify-bugs](https://expensify.slack.com/archives/C049HHMV9SM))
    - If it is, comment with a link to the original report, close the issue and add any novel details to the original issue instead
- [ ] This bug is reproducible using the reproduction steps in the OP. [S/O](https://stackoverflowteams.com/c/expensify/questions/14418#:~:text=The%20bug%20is%20reproducible)
    - If the reproduction steps are clear and you're unable to reproduce the bug, check with the reporter and QA first, then close the issue.
    - If the reproduction steps aren't clear and you determine the correct steps, please update the OP.
- [ ] This issue is filled out as thoroughly and clearly as possible
    - Pay special attention to the title, results, platforms where the bug occurs, and if the bug happens on staging/production.
- [ ] I have reviewed and subscribed to the linked Slack conversation to ensure Slack/Github stay in sync

**Comment by MelvinBot** [comment]
Job added to Upwork: https://www.upwork.com/jobs/~01823be2bad9582bf7

**Comment by MelvinBot** [comment]
Current assignee @flaviadefaria is eligible for the External assigner, not assigning anyone new.

**Comment by MelvinBot** [comment]
Triggered auto assignment to Contributor-plus team member for initial proposal review - @sobitneupane (`External`)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @pecanoro (`External`), see https://stackoverflow.com/c/expensify/questions/7972 for more details.

**Comment by khashayar-bo** [comment]
### Restating the Problem

HTML special characters are not rendered as plain text and are rather rendered to their corresponding character

### Proposal

There are two possible solutions: 

1. The sections that have the possibility to contain the special characters must have <xmp> html tag
2. When the user wants to save the data, the data should be parsed and if there is a character which is corresponding to an HTML special character, the value should be prefixed with &amp; so that when the value is interpreted by the browser, it would show as text and not the special character

**Comment by tienifr** [comment]
# Proposal

## Please re-state the problem that we are trying to solve in this issue.
The special html characters will be decoded in report and IOUs

## What is the root cause of that problem?

We're using `Str.htmlDecode` to convert text that makes `&nbsp` is converted to blank for example

## What changes do you think we should make in order to solve the problem?

We should remove `Str.htmlDecode` in some places like https://github.com/Expensify/App/blob/f40f9c9d81547292a53da4e48c23c2151a9d7fb7/src/pages/home/report/ReportActionItemFragment.js#L145, 
https://github.com/Expensify/App/blob/f40f9c9d81547292a53da4e48c23c2151a9d7fb7/src/components/ReportActionItem/IOUQuote.js#L73-L80,
and other places if needed

## Result

<img width="575" alt="Screen Shot 2023-02-15 at 18 45 35" src="https://user-images.githubusercontent.com/113963320/219018850-7ed8cc3c-7e7d-43c6-b287-f3e5a9ad34b8.png">

**Comment by tienifr** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
If the user name coincides with HTML entities like &nbsp, it's not showing as is in the chat.

### What is the root cause of that problem?
This is because we're unescaping HTML entities when displaying the user name in the chat. We should not do this because the user name should be shown as is.

### What changes do you think we should make in order to solve the problem?

Remove this logic `unescaping HTML entities when displaying the user name in the chat.` in https://github.com/Expensify/App/blob/50434b672aba8e9821c3f0e7968b1b1dc4124218/src/pages/home/report/ReportActionItemFragment.js#L95
Change it to display as is: `{(props.fragment.text)}`

### What alternative solutions did you explore? (Optional)
NA

**Reminder:** Please use plain English, be brief and avoid jargon. Feel free to use images, charts or pseudo-code if necessary. Do not post large multi-line diffs or write walls of text. Do not create PRs unless you have been hired for this job.

**Comment by dgaisan** [comment]
## Restating the Problem
There should be no blank space and the &nbsp should be shown normally

## Proposal
Right now the text(representing display name) that's being passed into Report component is explicitly html-escaped.
Instead we will need to render it as it (w/o escaping it).  It's already done this way(no escaping) in other places, for instance on the top of Settings component (right panel), right under the thumbnail

**Comment by eh2077** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.

Changing username to contain special characters, like &nbsp, makes App unable to display username correctly in chats and IOU messages.

### What is the root cause of that problem?

The root cause of this issue is that 

1. we don't check if the username input contains many special characters 
2. we save the username contains special characters to backend
3. in chats and IOU messages, we do html decode for username

### What changes do you think we should make in order to solve the problem?

To fix this issue, we can improve the validation rule of username. Currently, we only check `,` and `;` for username through this function

https://github.com/Expensify/App/blob/f839c5645addc5f5d85c05c5f82aa92819897c55/src/libs/ValidationUtils.js#L359-L367

We can improve it to check other special characters we don't want for username, like

```typescript
function isValidDisplayName(name) {
    const format = /[!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?]+/; // We can customize it for the App if needed
    return !format.test(name);
}
```

And update validation error hints. We can add similar checking at backend as well.

We can also use this method to check for other inputs that we don't want special characters.

### What alternative solutions did you explore? (Optional)

None

**Comment by sobitneupane** [comment]
Thanks for your proposal @khashayar-bo and @dgaisan.

Please do make use of [our proposal template](https://github.com/Expensify/App/blob/main/contributingGuides/PROPOSAL_TEMPLATE.md) to post proposals. 

Also it is good to use permalinks to explain root cause of the problem and the solution.

**Comment by sobitneupane** [comment]
Thanks for the proposal @tienifr.

[The proposal](https://github.com/Expensify/App/issues/15131#issuecomment-1431241018) looks good to me. 

@pecanoro First thing, I am not sure if it is really a bug. 

If we want to allow the use of special html characters like &nbsp and want it to be shown as it is, we should get rid of `str.htmlDecode` as suggested in the proposal. `str.htmlDecode` was being used from very beginning.

🎀👀🎀 C+ reviewed
cc: @pecanoro

**Comment by tienifr** [comment]
@eh2077 nice suggestion but I think it's more like a feature request.

With that change existing names that have the HTML entities will still show incorrectly.

Also users who use special characters in the name before will suddenly see their name showing error when they try to edit the name.

**Comment by eh2077** [comment]
Hi @tienifr, thanks for the comment.

I think what I proposed https://github.com/Expensify/App/issues/15131#issuecomment-1432414046 is the right fix if using html entities in username is unexpected from product perspective. I really don't think we would expect to use html tags in username initially, like having a username `<h1>David</h1>`

While if this is an expected feature, then I agree with you it'll be a feature request.

**Comment by flaviadefaria** [comment]
@sobitneupane @pecanoro any further thoughts here?

**Comment by pecanoro** [comment]
Taking a look at what it would be the desirable behaviour here so we can go with one proposal or another.

**Comment by pecanoro** [comment]
Ok, so yeah, to avoid potential weird injections, we should never decode the name and if we input `&nbsp`, we should show `&nbsp` as the name in the chats. I agree with @sobitneupane, @tienifr's proposal sounds good so assigning it to you.

**Comment by MelvinBot** [comment]
📣 @tienifr You have been assigned to this job by @pecanoro!
Please apply to this job in Upwork and leave a comment on the Github issue letting us know when we can expect a PR to be ready for review 🧑‍💻
Keep in mind: [Code of Conduct](https://github.com/Expensify/App/blob/main/CODE_OF_CONDUCT.md) | [Contributing](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md) 📖

**Comment by tienifr** [comment]
Hi @sobitneupane [This PR](https://github.com/Expensify/App/pull/15340) is ready for review, please help to review. Thanks

**Comment by MelvinBot** [comment]
📣 @tienifr! 📣

Hey, it seems we don’t have your contributor details yet! You'll only have to do this once, and this is how we'll hire you on Upwork.
Please follow these steps:

1. Get the email address used to login to your Expensify account. If you don't already have an Expensify account, create one [here](https://new.expensify.com/). If you have multiple accounts (e.g. one for testing), please use your main account email.
2. Get the link to your Upwork profile. It's necessary because we only pay via Upwork. You can access it by logging in, and then clicking on your name. It'll look like [this](https://www.upwork.com/freelancers/~01ca8ad8ce36b9eceb). If you don't already have an account, sign up for one [here](https://www.upwork.com/nx/signup/?dest=home).
3. Copy the format below and paste it in a comment on this issue. Replace the placeholder text with your actual details.

![Screen Shot 2022-11-16 at 4 42 54 PM](https://user-images.githubusercontent.com/30609178/202326034-1a292e64-48fd-4fef-9f4f-7feaaeedfd9f.png)

Format:
```
Contributor details
Your Expensify account email: <REPLACE EMAIL HERE>
Upwork Profile Link: <REPLACE LINK HERE>
```

**Comment by tienifr** [comment]
Contributor details
Your Expensify account email: christianwen18@gmail.com
Upwork Profile Link: https://www.upwork.com/freelancers/~01991fd5e5c11ef3ba

**Comment by MelvinBot** [comment]
✅ Contributor details stored successfully. Thank you for contributing to Expensify!

**Comment by MelvinBot** [comment]
⚠️ Looks like this issue was linked to a **Deploy Blocker** [here](https://github.com/Expensify/App/issues/15576)

If you are the assigned CME please investigate whether the linked PR caused a regression and leave a comment with the results.

If a regression has occurred and you are the assigned CM follow the instructions [here](https://stackoverflow.com/c/expensify/questions/13647).

If this regression could have been avoided please consider also proposing a recommendation to the PR checklist so that we can avoid it in the future.

**Comment by MelvinBot** [comment]
@pecanoro, @sobitneupane, @flaviadefaria, @tienifr Uh oh! This issue is overdue by 2 days. Don't forget to update your issues!

**Comment by pecanoro** [comment]
Not overdue, PR is on review. We deployed internal PR only a couple of days ago/

**Comment by tienifr** [comment]
@sobitneupane Please help me review [the PR](https://github.com/Expensify/App/pull/15340) since we have a small change. @pecanoro approved it but need your help to retest then she can merge the PR.

**Comment by pecanoro** [comment]
@s77rt Youn mentioned you were looking for more issues, so would you mind taking over the review for this one? It's almost ready but we need someone to final test all the changes. PR is [here](https://github.com/Expensify/App/pull/15340).

**Comment by sobitneupane** [comment]
I am reviewing the PR @pecanoro

**Comment by pecanoro** [comment]
Ohh got it haha. Then reverting back everything, sorry, since you didn't reply, I wasn't sure.

**Comment by MelvinBot** [comment]
📣 @sobitneupane You have been assigned to this job by @pecanoro!
Please apply to this job in Upwork and leave a comment on the Github issue letting us know when we can expect a PR to be ready for review 🧑‍💻
Keep in mind: [Code of Conduct](https://github.com/Expensify/App/blob/main/CODE_OF_CONDUCT.md) | [Contributing](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md) 📖

**Comment by MelvinBot** [comment]
@pecanoro, @sobitneupane, @flaviadefaria, @tienifr Uh oh! This issue is overdue by 2 days. Don't forget to update your issues!

**Comment by pecanoro** [comment]
Issue ended up being a bit more complex, so we needed a few small PRs to fix it, we are still working on it!

**Comment by MelvinBot** [comment]
@pecanoro, @sobitneupane, @flaviadefaria, @tienifr Uh oh! This issue is overdue by 2 days. Don't forget to update your issues!

**Comment by pecanoro** [comment]
I was on vacation for a few days, coming back to deal with the pending tasks.

**Comment by tienifr** [comment]
# Regression Test Proposal

**Bug**: Special html characters like &nbsp are not escaped in name and can bug out the UI

**Proposed Test Steps**:

1. Log in to account A and change the user name to &nbsp
2. Send some messages and send/request money to account B
3. Verify that the user name of account A is displayed correctly in all the reports, IOU messages and LHN in both accounts


Do we :+1: or :-1:

**Comment by sobitneupane** [comment]
@tienifr Looks like while resolving merge conflicts, you might have undone the change you made in SidebarUtils. Can you please create a PR to remove `Str.htmlDecode` from https://github.com/Expensify/App/blob/d8657bae27f7daa705803f21f2becce2f0b4e219/src/libs/SidebarUtils.js#L250

Please let me know if it is intentional.

cc: @pecanoro

**Comment by MelvinBot** [comment]
`Reviewing` label has been removed, please complete the "BugZero Checklist".

**Comment by MelvinBot** [comment]
The solution for this issue has been :rocket: deployed to production :rocket: in version 1.2.91-1 and is now subject to a 7-day regression period :calendar:. Here is the list of pull requests that resolve this issue:

- https://github.com/Expensify/App/pull/15340


If no regressions arise, payment will be issued on 2023-04-05. :confetti_ball:

After the hold period is over and BZ checklist items are completed, please complete any of the applicable payments for this issue, and check them off once done.
- [ ] External issue reporter
- [ ] Contributor that fixed the issue
- [ ] Contributor+ that helped on the issue and/or PR

As a reminder, here are the bonuses/penalties that should be applied for any External issue:
- Merged PR within 3 business days of assignment - 50% **bonus**
- Merged PR more than 9 business days after assignment - 50% **penalty**

**Comment by MelvinBot** [comment]
**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

- [ ] [@sobitneupane / @pecanoro] The PR that introduced the bug has been identified. Link to the PR:
- [ ] [@sobitneupane / @pecanoro] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
- [ ] [@sobitneupane / @pecanoro] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
- [ ] [@flaviadefaria] Determine if we should create a regression test for this bug.
- [ ] [@sobitneupane] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.
- [ ] [@flaviadefaria] Link the GH issue for creating/updating the regression test once above steps have been agreed upon:

**Comment by tienifr** [comment]
@sobitneupane After checking my PR https://github.com/Expensify/App/pull/15340/files I don't change anything in SidebarUtils. Pls correct me if I'm wrong. Thanks

**Comment by tienifr** [comment]
@sobitneupane After sending `&nbsp`, We encode the text both in lastMessageText of LHN and report comment to `&amp;nbsp`, but after receive the data from BE, lastMessageText is changed to `&nbsp`, report comment is still `&amp;nbsp`
<img width="240" alt="Screen Shot 2023-03-30 at 10 49 20" src="https://user-images.githubusercontent.com/113963320/228723946-a33c6fe1-54c8-4cc0-97e6-38b577aaaa6a.png">

Here is the result if I remove Str.htmlDecode:

https://user-images.githubusercontent.com/113963320/228724132-5fafc6a4-3fba-4085-a152-a917be4fe02c.mov

I suggest change as `lastMessageTextFromReport =  Str.htmlDecode(report ? report.lastMessageHtml : '')`

https://user-images.githubusercontent.com/113963320/228724519-702727a0-0feb-450a-98de-4f6bf6c7d090.mov

Please let me know what your thought is?

**Comment by sobitneupane** [comment]
@tienifr That's a good find.

Actually, issue lies in the following line:

https://github.com/Expensify/App/blob/75ba5f48afd06197da50f2d42c63dcd6e4cbd1ab/src/libs/ReportUtils.js#L900-L901

I am not sure why don't we directly use `text` instead of parsing text and converting html back to text. But I am afraid it might break things up. @tienifr Can you please test using `text` and see if it works.

**Comment by tienifr** [comment]
@sobitneupane Yes, we can't do that. Here's the result if I use `const textForNewComment = isAttachment ? CONST.ATTACHMENT_MESSAGE_TEXT: text`. I think we should change as I suggest here https://github.com/Expensify/App/issues/15131#issuecomment-1489645906. But it needs to change on BE side as well.

https://user-images.githubusercontent.com/113963320/228902660-cf404c08-2484-41be-b09c-bba3770dbaea.mov

**Comment by eh2077** [comment]
I found a regression about this PR see comment https://github.com/Expensify/App/pull/15340#discussion_r1154280552

Should we fix this before it's deployed to production?

cc @sobitneupane @tienifr  @pecanoro

**Comment by sobitneupane** [comment]
@tienifr Any update on https://github.com/Expensify/App/issues/15131#issuecomment-1488287488 and https://github.com/Expensify/App/pull/15340#discussion_r1154280552?

**Comment by tienifr** [comment]
@sobitneupane
Case 1: If we use `const textForNewComment = isAttachment ? CONST.ATTACHMENT_MESSAGE_TEXT: text` and remove `Str.htmlDecode` in https://github.com/Expensify/App/blob/d8657bae27f7daa705803f21f2becce2f0b4e219/src/libs/SidebarUtils.js#L250

`text` is not the same as `parser.htmlToText(htmlForNewComment)`. In the case user types `*abc*` text is `*abc*` but `parser.htmlToText(htmlForNewComment)` is `abc`

We render text in: https://github.com/Expensify/App/blob/df10eeba7106a2b7477208a8b4325d5b219d26af/src/pages/home/report/ReportActionItemFragment.js#L118-L122 we use RenderHTML to render htmlContent. If user types `*abc*` htmlContent is <strong>abc</strong> => bold abc will be rendered. But in LHN as we changed textForNewComment to text, that why it renders `*abc*` first. Please take a look at the video I attached above.

Case 2: My suggestion is wrong too if user types `*abc*` https://github.com/Expensify/App/issues/15131#issuecomment-1489645906

=> Should we return lastMessageText as encoded string from BE side, it will fix this problem

**Comment by sobitneupane** [comment]
I still believe we need to solve the issue here:

https://github.com/Expensify/App/blob/75ba5f48afd06197da50f2d42c63dcd6e4cbd1ab/src/libs/ReportUtils.js#L900-L901

**Comment by sobitneupane** [comment]
@tienifr I think we should do similar processing as done in backend so we get the same value as backend.

https://github.com/Expensify/App/blob/75ba5f48afd06197da50f2d42c63dcd6e4cbd1ab/src/libs/ReportUtils.js#L900-L901

**Comment by tienifr** [comment]
@sobitneupane @pecanoro Can you please help review my PR https://github.com/Expensify/App/pull/16957. It's fixed the bug in LHN and IOU. Thanks

**Comment by flaviadefaria** [comment]
I'm a bit confused, is this ready for payment? cc @pecanoro

**Comment by flaviadefaria** [comment]
I'm about to go OoO for a week so reassigning this.

**Comment by MelvinBot** [comment]
Triggered auto assignment to @zanyrenney (`Bug`), see https://stackoverflow.com/c/expensify/questions/14418 for more details.

**Comment by pecanoro** [comment]
@flaviadefaria There was a regression so we can't pay this one just yet.

**Comment by MelvinBot** [comment]
@pecanoro, @sobitneupane, @zanyrenney, @tienifr Uh oh! This issue is overdue by 2 days. Don't forget to update your issues!

**Comment by MelvinBot** [comment]
@pecanoro, @sobitneupane, @zanyrenney, @tienifr Still overdue 6 days?! Let's take care of this!

**Comment by zanyrenney** [comment]
How are we doing with the regression  @pecanoro ? Are we ready to pay this out now?

**Comment by MelvinBot** [comment]
`Reviewing` label has been removed, please complete the "BugZero Checklist".

**Comment by MelvinBot** [comment]
The solution for this issue has been :rocket: deployed to production :rocket: in version 1.3.1-3 and is now subject to a 7-day regression period :calendar:. Here is the list of pull requests that resolve this issue:

- https://github.com/Expensify/App/pull/16957
- https://github.com/Expensify/App/pull/17449


If no regressions arise, payment will be issued on 2023-04-26. :confetti_ball:

After the hold period is over and BZ checklist items are completed, please complete any of the applicable payments for this issue, and check them off once done.
- [ ] External issue reporter
- [ ] Contributor that fixed the issue
- [ ] Contributor+ that helped on the issue and/or PR

As a reminder, here are the bonuses/penalties that should be applied for any External issue:
- Merged PR within 3 business days of assignment - 50% **bonus**
- Merged PR more than 9 business days after assignment - 50% **penalty**

**Comment by MelvinBot** [comment]
**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

- [ ] [@sobitneupane] The PR that introduced the bug has been identified. Link to the PR:
- [ ] [@sobitneupane] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
- [ ] [@sobitneupane] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
- [ ] [@zanyrenney] Determine if we should create a regression test for this bug.
- [ ] [@sobitneupane] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.
- [ ] [@zanyrenney] Link the GH issue for creating/updating the regression test once above steps have been agreed upon:

**Comment by zanyrenney** [comment]
bump @pecanoro can you confirm the status of this one? I am not clear whether this is ready to pay out and I took this over when @flaviadefaria reassigned it so not clear here. Thanks!

**Comment by pecanoro** [comment]
@zanyrenney Yes, sorry! Now it's ready to be paid since it's past 2023-04-26. I think BugBot got confused with the timeline due to the regression. There was a regression after merging so take that into account when calculating the payouts and such 🤗

**Comment by pecanoro** [comment]
@zanyrenney Did it get paid?

**Comment by pecanoro** [comment]
Moving back to daily so we can this sort out soon.

**Comment by zanyrenney** [comment]
Had to create a new job post as Flavia's one was too old. https://www.upwork.com/jobs/~01ff4a14f72d218529

<img width="904" alt="2023-05-15_09-45-58" src="https://github.com/Expensify/App/assets/56830058/18a40374-3591-4efa-aed8-d8e446accba5">

Invited all. please accept invite for payout @tienifr @sobitneupane @esh-g

**Comment by jayeshmangwani** [comment]
@zanyrenney I am eligible for reporting [regression](https://github.com/Expensify/App/pull/16957/files#r1166301812) here, Can you please send me an offer for reporting [regression](https://expensify.slack.com/archives/C049HHMV9SM/p1681445959245769)?

**Comment by zanyrenney** [comment]
hey @jayeshmangwani - invited you.

**Comment by tienifr** [comment]
@zanyrenney I submitted proposal to the job, thanks!

**Comment by jayeshmangwani** [comment]
> hey @jayeshmangwani - invited you.

@zanyrenney I submitted a proposal for the job, thanks!

**Comment by zanyrenney** [comment]
Cool, thanks everyone. All the jobs have now been paid out. CLosing the issue!
